.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "plugins": [
+    "github"
+  ],
```

Comment:
Trying to see if I can make this `plugins` config optional by setting it in https://github.com/github/eslint-plugin-github/blob/master/lib/configs/recommended.js